base_external_api/views/res_partner_view.xml

```diff
@@ -20,4 +20,13 @@
         <field name="code">records.action_upload_partner_master()</field>
     </record>
 
+    <record id="action_download_ftp" model="ir.actions.server">
+        <field name="name">FTP Download Demo</field>
+        <field name="model_id" ref="base.model_res_partner" />
+        <field name="binding_model_id" ref="base.model_res_partner" />
+        <field name='groups_id' eval="[(4, ref('base.group_no_one'))]" />
+        <field name="state">code</field>
+        <field name="code">records.action_download_partner_master()</field>
```

Comment:
Maybe this should be in a demo data file.
In that case you could `records.action_download_partner_master(conn_code="DemoFTP")`

Reply:
Better yet, "records.action_download_partner_master(subdirectory="/Inbox", conn_code="DemoFTP")